### PR TITLE
an/SKY-12: Post create page design

### PR DESCRIPTION
**Pull request by anurmamedov**
Post create page designed

large size post-create page
medium size post-create page
small size post-create page

### CODE REVIEW

File: templates/post-create.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+    <body>
```
Comment (Aymammet) [inline templates/post-create.html:1]: can be removed

File: templates/post-create.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            </div>
+            <!-- Save and Cancel buttons -->
+            <div class="container-fluid">
+                <div class="text-center">
+                    <button type="button" class="btn btn-secondary mb-4 col-5">Cancel</button>
+                    <button type="button" class="btn btn-primary mb-4 col-5">Create</button>
+                </div>
+            </div>
+        </div>
+    </body>
```
Comment (Aymammet) [inline templates/post-create.html:27]: can be removed for now

**Review by Aymammet — CHANGES_REQUESTED** [review]

File: templates/post-create.html (hunk 1 above)
Comment (anurmamedov) [reply templates/post-create.html]: removed

File: templates/post-create.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                <input class="form-control " type="file" id="formFile">
+            </div>
+            <!-- Save and Cancel buttons -->
+            <div class="container-fluid">
+                <div class="text-center">
+                    <button type="button" class="btn btn-secondary mb-4 col-5">Cancel</button>
+                    <button type="button" class="btn btn-primary mb-4 col-5">Create</button>
+                </div>
+            </div>
+        </div>
```
Comment (Aymammet) [inline templates/post-create.html:25]: your inputs should be in form tag.

**Review by Aymammet — CHANGES_REQUESTED** [review]

File: templates/post-create.html (hunk 3 above)
Comment (anurmamedov) [reply templates/post-create.html:25]: it is from line 2 to  7. or do you mean the whole page ?

Comment (Aymammet) [reply templates/post-create.html:25]: Your input elements are outside of form tag. 
image

**Review by Aymammet — CHANGES_REQUESTED** [review]

**Review by Aymammet — APPROVED** [review]

**Pull request merged**